.binder/postBuild

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# update pip
+pip install --upgrade pip
+
+# display installed versions
+python -V
+pip list
+
+# install NLTL Punkt tokenizer
+python -m nltk.downloader punkt
+
+# install SpaCy language model
+python -m spacy download de_dep_news_trf
```

Comment:
To solve this, maybe load a smaller model? Or is the smaller model not accurate enough? 
```suggestion
python -m spacy download de_core_news_sm
```

Reply:
See #19

Reply:
The binder config is a great improvement, however, I would not aim for having everything pre-loaded/installed. The notebook in question is more of a demonstrator that shows how to do such analysis, and the approach involves loading a model. Theoretically, someone might want to change the model and use a different language corpus, then, he or she would have to load a different model anyways. Maybe we focus on the minimum requirements to do DraCor research, e.g. pre-install pydracor and some auxillary packages. SpaCy would be good, but maybe leave the model out of the pre-install and document that in the notebook instead?

Reply:
I agree. Pre-loading all potential models one might use for an analysis just blows up the docker image size for every user and their startup time.